test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightContextUriParserTests.cs

```diff
@@ -31,13 +31,21 @@ private EdmModel GetModel()
             return model;
         }
 
-        // TODO: Support relative context uri and resolving other relative uris
         [Fact]
-        public void ParseRelativeContextUrlShouldThrowException()
+        public void ParseRelativeContextUrlShouldNotThrowException()

```

Comment:
In the PR description you mention that relative context URLs of the form: `@context: "Products"` should be supported, but there are no tests to verify this scenario.